src/assets/i18n/fr_FR.json

```diff
@@ -650,6 +650,16 @@
   "checkout.credit_card.user.lastname.missing.error": "Veuillez entrer le nom de famille du titulaire de la carte.",
   "checkout.credit_card.user.name.missing.error": "Veuillez entrer le nom du titulaire de la carte.",
   "checkout.credit_card.validityTime.error.notFound": "La durée de validité maximale du CVC n’a pas été trouvée.",
+  "checkout.desired_delivery_date.apply.button.label": "Appliquer",
+  "checkout.desired_delivery_date.error.date": "La date n’est pas valide. Veuillez utiliser le format correct (jj-mm-aaaa).",
```

Comment:
Please check if the date picker plugin has localized date formats. Else the given format in our messages (lines 654 and 660) has to be changed accordingly.